pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/participant/DownloadParticipantListPayload.java

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.dsm.model.participant;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.google.gson.annotations.SerializedName;
+import lombok.Getter;
+import lombok.Setter;
+import org.broadinstitute.dsm.model.ParticipantColumn;
+
+@Getter
+@Setter
+public class DownloadParticipantListPayload {
```

Comment:
I think this class should be in `org.broadinstitute.dsm.model.excel` and not `org.broadinstitute.dsm.model.participant`, since it belongs to the exporting excel feature.